nixos/modules/services/misc/languagetool.nix

```diff
@@ -0,0 +1,82 @@
+{ config, lib, options, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.languagetool;
+in {
+  options.services.languagetool = {
+    enable = mkOption {
+      type = types.bool;
+      example = true;
+      default = false;
+      description = ''
+        Start the HTTP LanguageTool server.
+      '';
```

Comment:
please write `mdDoc` descriptions instead, that will make the markdown migration much easier. thanks!

Reply:
I'm not familiar with 'mDoc'. Can you point me to an example or documentation?

Reply:
it's a marker function that tells the documentation system to render the following string using [nixpkgs markdown rules](https://nixos.org/manual/nixpkgs/stable/#sec-contributing-markup) instead of docbook. in the case of this option just adding the marker would be sufficient :)

Reply:
I've added it to all `documentation` attributes.